src/NavigationState.ts

```diff
@@ -74,6 +76,12 @@ Navigation.events().registerCommandListener((name, params) => {
   if (params.commandId && name === CommandName.ShowOverlay) {
     overlayWasShown = true;
   }
+
+  if (name === CommandName.SetRoot) {
+    const initialTabIndex = params?.layout?.root?.data?.options?.bottomTabs?.currentTabIndex ?? 0;
+    state.initialTabIndex = initialTabIndex;
```

Comment:
Why do you need initialTabIndex
now that you set activeTabIndex you can use it when the app loads, no?

Reply:
yes. but i thought that maybe someone will want it..
i don't mind removing it, W/E you think is best

Reply:
seems like its redundant